### ISSUE TITLE
Moving all hardcoded strings to strings.xml

### ISSUE THREAD

**Issue by opticod** [BuildmLearn/BuildmLearn-Toolkit-Android#279]
You should be moving all the existing necessary hardcoded strings to strings.xml.
Anyone want to help me out here?

**Comment by octacode** [comment]
Count me in.

**Comment by codingblazer** [comment]
If still need help, I will be happy to help.

**Comment by rijuldhir** [comment]
I can help too.

**Comment by codingblazer** [comment]
@opticod I think there are too many lint issues also. We can add those too if you feel so.

**Comment by opticod** [comment]
@octacode @codingblazer @rijuldhir , Cool, then lets distribute these works among us all. 
@octacode Can you fix all hard coded strings in first 10 folders(alphabetically sorted order)? viz. activity to matchtemplate folder.
@rijuldhir Can you fix all hard coded strings in last 7 folders(alphabetically sorted order)? viz. model to views folder. + ToolkitApplication + all files in res folder if any.
@codingblazer Can you fix all lint issues except the "HARDCODED String" issue ?
 
Guys let me know, if anyone wants any modification.

**Comment by octacode** [comment]
Awesome @opticod. ^^

**Comment by octacode** [comment]
@opticod. We only need to hard code those strings which we have to display as a message to the user. Right?

**Comment by vishwesh3** [comment]
If still there is any unassigned issue, let me know I will be happy to work on that.

**Comment by opticod** [comment]
@octacode Yes, and rest wherever possible but not all declare them as constant globally.

**Comment by opticod** [comment]
@vishwesh3 Cool, Can you replace all hardcoded dimes into dimen.xml ?

**Comment by opticod** [comment]
Guys when replacing them please minimise the redundancy of newly added strings or dimen,  by this what i mean is ...if there already an existing entry of it then better use that rather than adding one more entry in xml file.

**Comment by vishwesh3** [comment]
Yes.

**Comment by opticod** [comment]
I realised that, most of you are creating minute mistakes while making PRs but they can be critical in production, so will not merge any big change before DEC-2016 release.

**Comment by sm86** [comment]
Is the issue closed? Can I help in any way

**Comment by opticod** [comment]
@sm86, this issues is already assigned.
